dissect/target/plugins/apps/webservers/caddy.py

```diff
@@ -0,0 +1,84 @@
+import re
+from zoneinfo import ZoneInfo, ZoneInfoNotFoundError
+from datetime import datetime, timezone
+
+from dissect.target import plugin
+from dissect.target.plugins.apps.webservers.webservers import WebserverRecord
+
+
+class CaddyPlugin(plugin.Plugin):
+
+    __namespace__ = "caddy"
+
+    def __init__(self, target):
+        super().__init__(target)
+
+        self.LOG_FILE_PATH = self.get_log_path()
+        self.target_timezone = target.datetime.tzinfo
+
+    @plugin.internal
+    def get_log_path(self):
+
+        # Try the default log location
+        log_file = "/var/log/caddy_access.log"
+        if log_file.exists():
+            self.target.log.debug(f"Log file found at {self.LOG_FILE_PATH}")
+            return log_file
```

Comment:
Would you not rather combine this with possible output of parsing the config file?